locust/locustfile.py

```diff
@@ -0,0 +1,67 @@
+# SPDX-License-Identifier: Apache-2.0
+# SPDX-FileCopyrightText: © 2024 Tenstorrent AI ULC
+
+import json
+import os
+
+import jwt
+
+from data_reader import DataReader
+from locust import FastHttpUser, events, task
+
+# Constants for timeouts and API configuration
+NETWORK_TIMEOUT = 300.0
+CONNECTION_TIMEOUT = 300.0
+API_ENDPOINT = "/v1/completions"
+DEFAULT_PARAMS = {
+    "model": "meta-llama/Llama-3.1-70B-Instruct",
+    "temperature": 1.0,
+    "top_k": 10,
+    "top_p": 0.9,
+}
+
+# Global variable to store data iterator
+data_iter = None
+
+def get_authorization():
+    authorization = os.getenv("AUTHORIZATION", None)
+    if authorization is None:
+        jwt_secret = os.getenv("JWT_SECRET", None)
+        if jwt_secret is None:
+            raise ValueError(
+                "Neither AUTHORIZATION or JWT_SECRET environment variables are set."
+            )
+        json_payload = json.loads('{"team_id": "tenstorrent", "token_id":"debug-test"}')
+        encoded_jwt = jwt.encode(json_payload, jwt_secret, algorithm="HS256")
+        authorization = f"{encoded_jwt}"
+    return authorization
+
+
+# Event listener to load custom data before tests start
+@events.test_start.add_listener
+def load_custom_data(**kwargs):
+    global data_iter
+    data_iter = DataReader()
+
+
+class ServeUser(FastHttpUser):
+    # Set test parameters
+    network_timeout = NETWORK_TIMEOUT
+    connection_timeout = CONNECTION_TIMEOUT
+    headers = {"Authorization": get_authorization()}
```

Comment:
```suggestion
    headers = {"Authorization": f"Bearer {get_authorization()}"}
```

Reply:
With adding "Bearer " to the Auth header locust tests ran fine with JWT auth turned on for me.

Reply:
I kept getting "unauthorized" with Bearer. I need to check which I was using, jwt or auth. But, can you test out both and recommend which method we should stick to?